WeatherApi

```diff
@@ -1 +1,41 @@
 
+using Microsoft.Data.SqlClient; 
+using Newtonsoft.Json.Linq; 
+using System; 
+using System.Net; 
+namespace WeatherApi
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
This weather client is really badly written. I would re-write the whole thing with the suggestions below. Also the server side that it is downloading weather from has major security issues, please consider getting the weather from an alternative API.